providers/netcore/src/lib/netcore.provider.ts

```diff
@@ -71,4 +84,61 @@ export class NetCoreProvider implements IEmailProvider {
       code: CheckIntegrationResponseEnum.SUCCESS,
     };
   }
+
+  getMessageId(body: any | any[]): string[] {
+    if (Array.isArray(body)) {
+      return body.map((item) => item.TRANSID);
+    }
+
+    return [body.TRANSID];
+  }
+
+  parseEventBody(
+    body: any | any[],
+    identifier: string
+  ): IEmailEventBody | undefined {
+    if (Array.isArray(body)) {
+      body = body.find((item) => item.TRANSID === identifier);
+    }
+
+    if (!body) {
+      return undefined;
+    }
+
+    const status = this.getStatus(body.EVENT);
+
+    if (status === undefined) {
+      return undefined;
+    }
+
+    return {
+      status: status,
+      date: new Date().toISOString(),
```

Comment:
the bodu does not contain a date we can use?

Reply:
Here is a sample webhook from the docs:

[ 	{ 			
     "TRANSID": 15536250111702803, 			"RESPONSE": " - 2.0.0 (success)", 			"EMAIL": "test@gmail.com", 			"TIMESTAMP": 1553681625, 			"FROMADDRESS": "info@mydomain.com", 			"EVENT": "sent", 	
     "MSIZE": 2155,
     "X-APIHEADER": "UNIQUEID", 
     "TAGS": "mytag1" 	
} ]

we do have a timestamp here but no date.

Reply:
Okay then we keep it like you have it now 👍

Reply:
I think we can do that by passing the timestamp to Date(). Correct me if I'm wrong.